code/modules/antagonists/wraith/objs/item/spirit_candle.dm

```diff
@@ -23,12 +23,21 @@
 		if (src.on)
 			src.visible_message("<span class='notice'>[user] blows on [src], its eyes emit a threatening glow!</span>")
 			for(var/mob/wraith/W in orange(4, user))
-				if ((W.last_spirit_candle_time + (W.forced_haunt_duration * 2)) < TIME)
-					W.last_spirit_candle_time = TRUE
+				//Small grace period to run away after being manifested if you managed to survive so you dont get chain-manifested
+				if ((W.last_spirit_candle_time + (W.forced_haunt_duration + 6 SECONDS)) < TIME)
+					W.last_spirit_candle_time = TIME
 					W.setStatus("corporeal", W.forced_haunt_duration, TRUE)
 					boutput(W, "<span class='alert'>A malignant spirit pulls you into the physical world! You begin to gather your forces to try and escape to the spirit realm...</span>")
 				else
 					boutput(user, "<span class='notice'>[src] vibrates slightly in your hand. A hostile entity lurks nearby but resisted our attempts to reveal it!</span>")
+			for(var/mob/wraith/poltergeist/P in orange(4, user))
+				if ((P.last_spirit_candle_time + (P.forced_haunt_duration + 6 SECONDS)) < TIME)
+					P.last_spirit_candle_time = TIME
+					P.setStatus("corporeal", P.forced_haunt_duration, TRUE)
+					boutput(P, "<span class='alert'>A malignant spirit pulls you into the physical world! You begin to gather your forces to try and escape to the spirit realm...</span>")
+				else
+					boutput(user, "<span class='notice'>[src] vibrates slightly in your hand. A hostile entity lurks nearby but resisted our attempts to reveal it!</span>")
```

Comment:
the above for loop is already grabbing them, they're of type wraith

Reply:
Indeed, i seem to have completely glossed over that part. I'll adapt the pr.